### PR TITLE
Update registrering filiters

### PR DESCRIPTION
**Pull request by dragutin-nav**
## Describe your changes

We have few filters that are focused on registrering data (utdaning, brukers situasjon) which didn't have option to show users who are missing those data. Option to show missing data for registering will provide full picture of veileder portfolio, so veiledere can see which brukers are missing registrering data and eventually do something about that. 

## Trello ticket number and link

https://trello.com/c/9JuUHsVH

## Type of change

- [✔️] New feature (non-breaking change which adds functionality)

## Checklist before requesting a review
- [✔️] I have performed a self-review of my code
- [✔️] If it is a core feature, I have added thorough tests.
- [❌ ] We need to implement grafana analytics

### CODE REVIEW

File: src/main/java/no/nav/pto/veilarbportefolje/domene/filtervalg/DinSituasjonSvarDto.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package no.nav.pto.veilarbportefolje.domene.filtervalg;
+
+public enum DinSituasjonSvarDto {
```
Comment (slovrid) [inline src/main/java/no/nav/pto/veilarbportefolje/domene/filtervalg/DinSituasjonSvarDto.java:3]: I don't think it is necessary to mix the "DTO" concept into enums and their names. 
```suggestion
public enum DinSituasjonSvar {
```

File: src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchQueryBuilder.java [hunk 2]
```diff
@@ -212,6 +212,58 @@ static void leggTilManuelleFilter(BoolQueryBuilder queryBuilder, Filtervalg filt
         if (filtervalg.harEnsligeForsorgereFilter() && filtervalg.getEnsligeForsorgere().contains(EnsligeForsorgere.OVERGANGSSTØNAD)) {
             queryBuilder.must(existsQuery("enslige_forsorgere_overgangsstonad"));
         }
+
+        if (filtervalg.harDinSituasjonSvar()) {
+            BoolQueryBuilder brukerensSituasjonSubQuery = boolQuery();
+            filtervalg.registreringstype.forEach(dinSituasjonSvar -> {
+                if (dinSituasjonSvar.equals(DinSituasjonSvarDto.INGEN_DATA)) {
```
Comment (slovrid) [inline src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchQueryBuilder.java:219]: Is it necessary to use `.equals` when comparing to enum values?

Comment (dragutin-nav) [reply src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchQueryBuilder.java:219]: Will fix.

File: src/main/java/no/nav/pto/veilarbportefolje/domene/filtervalg/DinSituasjonSvarDto.java (hunk 1 above)
Comment (dragutin-nav) [reply src/main/java/no/nav/pto/veilarbportefolje/domene/filtervalg/DinSituasjonSvarDto.java]: @slovrid here er requested changes: https://github.com/navikt/veilarbportefolje/pull/1126/commits/9b37f6d4fc33f6c04f531223edbaab43faa2fa4d

**Review by slovrid — APPROVED** [review]

**Pull request merged**